discovery-infra/download_logs.py

```diff
@@ -164,7 +164,10 @@ def download_logs(client: InventoryClient, cluster: dict, dest: str, must_gather
                 recreate_folder(os.path.join(output_folder, "must-gather"))
                 config_etc_hosts(cluster['name'], cluster['base_dns_domain'],
                                  helper_cluster.get_api_vip_from_cluster(client, cluster))
-                download_must_gather(kubeconfig_path, os.path.join(output_folder, "must-gather"))
+                try:
```

Comment:
don't catch it here but inside download_must_gather

Reply:
OK, done